coalib/bearlib/spacing/SpacingHelper.py

```diff
@@ -25,19 +25,12 @@ def get_indentation(self, line: str):
         :param line: A string to check for indentation.
         :return:     The indentation count in spaces.
         """
-        count = 0
-        for char in line:
-            if char == ' ':
-                count += 1
-                continue
 
-            if char == '\t':
-                count += self.tab_width - (count % self.tab_width)
-                continue
+        line_expanded = line.expandtabs(self.tab_width)
```

Comment:
where are you counting the number of spaces?

Reply:
In the line with  
```python
 indentation_count = len(line_expanded) - len(line_expanded.lstrip())
```

*Subtracting the amount of characters in a line with the amount
of characters of this same line without the spaces on the left side,
we can get the count of indentation of this line.*